android/src/main/java/com/bitmovin/player/reactnative/OfflineModule.kt

```diff
@@ -58,11 +59,11 @@ class OfflineModule(private val context: ReactApplicationContext) : ReactContext
      */
     @ReactMethod
     fun initWithConfig(nativeId: NativeId, config: ReadableMap?, drmNativeId: NativeId?, promise: Promise) {
-        uiManager()?.addUIBlock {
+        context.uiManagerModule?.addUIBlock {
```

Comment:
Why isn't this class a BitmovinBaseModule with the same resolveOnUIThread logic?

Reply:
Is there a case where config can actually be null?